sources/packages/api/src/services/application/application.service.ts

```diff
@@ -952,9 +982,9 @@ export class ApplicationService extends RecordDataModelService<Application> {
   async getApplicationDetailsForCOE(
     locationId: number,
     applicationId: number,
-    requiredCOEApplication: boolean = false,
```

Comment:
why is argument type(boolean) removed?

Reply:
![image](https://user-images.githubusercontent.com/77353155/135092341-685608fe-1602-4279-b703-a58ace1cc0bf.png)

Reply:
FYI. Here is more detail about why es-lint is discouraging it.
![image](https://user-images.githubusercontent.com/54600590/135116883-d7f30e37-0818-4b7f-b265-071e48687c64.png)